README.md

```diff
@@ -68,7 +68,19 @@ engines:
 Note that you have the update the YAML structure under the `languages` key to
 the Hash type to support extra configuration.
 
+### Excluding files and directories
+
+As with any other Codeclimate engine, you can exclude certain files or directories from being analyzed. For more information, [see *Exclude paths for specific engines* on our documentation][exclude-files-engine].
```

Comment:
Couple nits:

"Code Climate", not "Codeclimate". Can you hard-wrap this at 80 columns, like the rest of the document? I think just the bit you've emphasized (Exclude paths for specific engines) should be the link (rather than emphasizing it within a bigger link).